plugins/field-bitmap/src/field-bitmap.ts

```diff
@@ -0,0 +1,595 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Bitmap input field.
```

Comment:
This fileoverview isn't adding anything and we no longer include author annotations so I would remove this comment

Reply:
Done!